### DISCUSSION TITLE
有批量发帖的插件吗？

### DISCUSSION THREAD

**Discussion by itxwh** [MoeNetwork/Tieba-Cloud-Sign#241] [疑问] [answered]
编辑好标题和内容后给批量给多个贴吧发帖的插件有吗？

**Comment by n0099** [comment]
https://github.com/96dl/Tieba-Cloud-Sign-Plugins/tree/master/ver4_post

**Reply by itxwh** [reply]
这个我看了，只能回帖，不能发帖

**Comment by BANKA2017** [comment, marked as answer]
有，但仅限**有**，不保能用。

**Reply by itxwh** [reply]
哪个插件，我看看

**Reply by BANKA2017** [reply]
就是上面那个，用了旧版的接口所以用不了，而云签本体是无法处理用了protobuf的新版接口，所以还需要另请高明了
